egs/fame/s5/local/score_combine.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
Are you combining the results now? If not, please remove. Generally, aim at removing all unused scripts in local/